### PR TITLE
Fix upgrade from kube-up

### PR DESCRIPTION
**Pull request by justinsb**
Some of the validation & changes we introduced broke the kube-up upgrade.

### CODE REVIEW

File: upup/pkg/kutil/delete_cluster.go [hunk 1]
```diff
@@ -827,6 +821,21 @@ func ListRouteTables(cloud fi.Cloud, clusterName string) ([]*ResourceTracker, er
 	return trackers, nil
 }
 
+func DescribeRouteTables(cloud fi.Cloud) ([]*ec2.RouteTable, error) {
+	c := cloud.(awsup.AWSCloud)
+
+	glog.V(2).Infof("Listing EC2 RouteTables")
+	request := &ec2.DescribeRouteTablesInput{
```
Comment (chrislovecnm) [inline upup/pkg/kutil/delete_cluster.go]: Does this return err that we can check?

**Comment by chrislovecnm** [comment]
@justinsb any candidates for unit testing on this?

**Comment by chrislovecnm** [comment]
We should at least have a unit test that an imported cluster name doesn’t fail validation

And maybe that the route table name matches, though that one is just another reason I want to move the models into code

**Comment by justinsb** [comment]
@chrislovecnm we have a user trying to use this - can we merge and I'll a unit test later?

**Comment by chrislovecnm** [comment]
LGTM

**Pull request merged**